src/App.tsx

```diff
@@ -1,10 +1,74 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { Notification } from './components/Notification/Notification';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+
+const USER_ID = 10140;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
+
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState<string>('all');
+  const [itemsLeft, setItemsLeft] = useState<number>(0);
+  const [itemsCompleted, setitemsCompleted] = useState<number>(0);
+
+  const countItemsLeft = todos.filter(todo => !todo.completed).length
+  const countItemsCompleted = todos.filter(todo => todo.completed).length
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then((fetchedTodos: Todo[]) => {
+        setTodos(fetchedTodos);
+      })
+      .catch((fetchedError: Error) => {
+        setError(fetchedError?.message ?? 'Something went wrong');
+      });
+  }, []);
+
+  useEffect(() => {
+    let timeout: NodeJS.Timeout;
+
+    if (error) {
+      timeout = setTimeout(() => {
+        setError(null);
+      }, 3000);
+    }
+
+    return () => {
+      clearTimeout(timeout);
+    };
+  }, [error]);
+
+  const filteredByStatus = (todosFromServer: Todo[], StateStatus: string) => {
+    switch (StateStatus) {
```

Comment:
Consider renaming StateStatus to stateStatus to follow camelCase naming convention.